src/wallet/wallet.ts

```diff
@@ -147,6 +158,44 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     await handleCreate(data.status);
   }
 
+  /**
+   * onUpdateTx: Event called when a transaction is updated
+   *
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  async onUpdateTx(updatedTx) {
```

Comment:
Why is it async?

Reply:
It shouldn't be! Fixed in a483314d5ded017d66166fa8a282c23e98260626